tests/e2e/utils/common.go

```diff
@@ -134,424 +128,86 @@ type ServicebusResponse struct {
-					}, {
-						Name:            "dind-daemon",
-						SecurityContext: &v1.SecurityContext{Privileged: &IsSecureCtx},
-						Image:           "docker:dind",
-						Resources: v1.ResourceRequirements{
-							Requests: v1.ResourceList{
-								v1.ResourceCPU:    resource.MustParse("20m"),
-								v1.ResourceMemory: resource.MustParse("256Mi"),
-							},
-						},
-						VolumeMounts: []v1.VolumeMount{{Name: "docker-graph-storage", MountPath: "/var/lib/docker"}},
+			Template: v1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"app":                 name,
+						constants.E2ELabelKey: constants.E2ELabelValue,
 					},
 				},
-				NodeSelector: map[string]string{"k8snode": "kb-perf-node"},
-				Volumes: []v1.Volume{
-					{Name: "cert", VolumeSource: v1.VolumeSource{HostPath: &v1.HostPathVolumeSource{Path: "/etc/kubeedge/certs"}}},
-					{Name: "conf", VolumeSource: v1.VolumeSource{ConfigMap: &v1.ConfigMapVolumeSource{LocalObjectReference: v1.LocalObjectReference{Name: configmap}}}},
-					{Name: "docker-graph-storage", VolumeSource: v1.VolumeSource{EmptyDir: &v1.EmptyDirVolumeSource{}}},
-				},
-			},
-		},
-	}
-	return &deplObj
-}
-
-// Function to create cloudcore deploymentspec object
-func cloudcoreDeploymentSpec(imgURL, configmap string, replicas int) *apps.DeploymentSpec {
-	portInfo := []v1.ContainerPort{{ContainerPort: 10000, Protocol: "TCP", Name: "websocket"}, {ContainerPort: 10001, Protocol: "UDP", Name: "quic"}}
-
-	deplObj := apps.DeploymentSpec{
-		Replicas: func() *int32 { i := int32(replicas); return &i }(),
-		Selector: &metav1.LabelSelector{MatchLabels: map[string]string{"app": "cloudcore"}},
-		Template: v1.PodTemplateSpec{
-			ObjectMeta: metav1.ObjectMeta{
-				Labels: map[string]string{"app": "cloudcore"},
-			},
-			Spec: v1.PodSpec{
-				HostNetwork:   true,
-				RestartPolicy: "Always",
-				Containers: []v1.Container{
-					{
-						Name:            "cloudcore",
-						Image:           imgURL,
-						ImagePullPolicy: v1.PullPolicy("IfNotPresent"),
-						Resources: v1.ResourceRequirements{
-							Requests: v1.ResourceList{
-								v1.ResourceCPU:    resource.MustParse("100m"),
-								v1.ResourceMemory: resource.MustParse("512Mi"),
-							},
+				Spec: v1.PodSpec{
+					Containers: []v1.Container{
+						{
+							Name:  name,
+							Image: imgURL,
 						},
-						Ports:        portInfo,
-						VolumeMounts: []v1.VolumeMount{{Name: "cert", MountPath: "/etc/kubeedge/certs"}, {Name: "conf", MountPath: "/etc/kubeedge/cloud/conf"}},
 					},
-				},
-				Volumes: []v1.Volume{
-					{Name: "cert", VolumeSource: v1.VolumeSource{HostPath: &v1.HostPathVolumeSource{Path: "/etc/kubeedge/certs"}}},
-					{Name: "conf", VolumeSource: v1.VolumeSource{ConfigMap: &v1.ConfigMapVolumeSource{LocalObjectReference: v1.LocalObjectReference{Name: configmap}}}},
+					NodeSelector: map[string]string{
+						"node-role.kubernetes.io/edge": "",
+					},
```

Comment:
I'd suggest to document this somewhere, saying, how we run the e2e testcase after this change, so we can just follow the doc instead of diving into the source.

Reply:
users do not aware this change and in the follow pr, I would add doc to instruct how to run e2e

Reply:
+1